<html lang="en">
  <head>
    <!-- Required meta tags -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="shortcut icon" href="https://static.igem.wiki/common/icons/favicons/igem-2022.svg"/>
    <link rel="license" href="https://creativecommons.org/licenses/by/4.0/"/>

    <!-- Bootstrap CSS -->
<!--    <link href="{{ url_for('static', filename = 'bootstrap.min.css') }}" rel="stylesheet"> -->
<link href="../../static/bootstrap.min.css" rel="stylesheet">
    <!-- Custom CSS -->
<!--    <link href="{{ url_for('static', filename = 'style.css') }}" rel="stylesheet">-->
<link href="../../static/style.css" rel="stylesheet">
<link href="../../static/style/modal.css" rel="stylesheet">

    <title>engineering | IISER_TVM - iGEM 2022</title>

</head>

<body style="background-color: #fdfff0">

<nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top">
  <div class="container">

    <!---- TEAM NAME ---->
    <a class="navbar-brand" href="#">IISER_TVM</a>

    <!---- SMALL SCREEN MENU ICON ---->
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div class="collapse navbar-collapse" id="navbarSupportedContent">
      <ul class="navbar-nav ml-auto left-aligned">

        <!---- HOME ---->
        <li class="nav-item">
          <a class="nav-link" href="{{ url_for('index') }}">Home</a>
        </li>

        <!---- TEAM ---->
        <li class="nav-item dropdown">
          <a class="nav-link dropdown-toggle" href="{{ url_for('pages', page='team') }}" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
            Team
          </a>
          <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
            <li><a class="dropdown-item" href="{{ url_for('pages', page='team') }}">Team</a></li>
            <li><a class="dropdown-item" href="{{ url_for('pages', page='attributions') }}">Attributions</a></li>
            <li><a class="dropdown-item" href="{{ url_for('pages', page='collaborations') }}">Collaborations</a></li>
          </ul>
        </li>

        <!---- PROJECT ---->
        <li class="nav-item dropdown">
          <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
            Project
          </a>
          <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
            <li><a class="dropdown-item" href="{{ url_for('pages', page='communication') }}">Communication</a></li>
            <li><a class="dropdown-item" href="{{ url_for('pages', page='contribution') }}">Contribution</a></li>
            <li><a class="dropdown-item" href="{{ url_for('pages', page='description') }}">Description</a></li>
            <li><a class="dropdown-item" href="{{ url_for('pages', page='engineering') }}">Engineering</a></li>
            <li><a class="dropdown-item" href="{{ url_for('pages', page='experiments') }}">Experiments</a></li>
            <li><a class="dropdown-item" href="{{ url_for('pages', page='implementation') }}">Implementation</a></li>
            <li><a class="dropdown-item" href="{{ url_for('pages', page='notebook') }}">Notebook</a></li>
            <li><a class="dropdown-item" href="{{ url_for('pages', page='partnership') }}">Partnership</a></li>
            <li><a class="dropdown-item" href="{{ url_for('pages', page='proof-of-concept') }}">Proof of Concept</a></li>
            <li><a class="dropdown-item" href="{{ url_for('pages', page='results') }}">Results</a></li>
          </ul>
        </li>

        <!---- PARTS ---->
        <li class="nav-item dropdown">
          <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
            Parts
          </a>
          <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
            <li><a class="dropdown-item" href="{{ url_for('pages', page='part-collection') }}">Part Collection</a></li>
            <li><a class="dropdown-item" href="{{ url_for('pages', page='parts') }}">Parts</a></li>
            <li><a class="dropdown-item" href="{{ url_for('pages', page='improve') }}">Improve</a></li>
          </ul>
        </li>

        <!---- SAFETY ---->
        <li class="nav-item">
          <a class="nav-link" href="{{ url_for('pages', page='safety') }}">Safety</a>
        </li>

        <!---- HUMAN PRACTICES ---->
        <li class="nav-item">
          <a class="nav-link" href="{{ url_for('pages', page='human-practices') }}">Human Practices</a>
        </li>

        <!---- AWARDS ---->
        <li class="nav-item dropdown">
          <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
            Awards
          </a>
          <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
            <li><a class="dropdown-item" href="{{ url_for('pages', page='education') }}">Education</a></li>
            <li><a class="dropdown-item" href="{{ url_for('pages', page='entrepreneurship') }}">Entrepreneurship</a></li>
            <li><a class="dropdown-item" href="{{ url_for('pages', page='hardware') }}">Hardware</a></li>
            <li><a class="dropdown-item" href="{{ url_for('pages', page='inclusivity') }}">Inclusivity</a></li>
            <li><a class="dropdown-item" href="{{ url_for('pages', page='measurement') }}">Measurement</a></li>
            <li><a class="dropdown-item" href="{{ url_for('pages', page='model') }}">Model</a></li>
            <li><a class="dropdown-item" href="{{ url_for('pages', page='plant') }}">Plant</a></li>
            <li><a class="dropdown-item" href="{{ url_for('pages', page='software') }}">Software</a></li>
            <li><a class="dropdown-item" href="{{ url_for('pages', page='sustainable') }}">Sustainable</a></li>
          </ul>
        </li>

      </ul>
    </div>
  </div>
</nav>

<!-- Page Content -->
<header class="masthead" style= background:url("../../static/images/woman.png");>
</header>

<div class="cointainer my-4 mx-4">
  <div class="row">

    <div class="col-3">
      <ul class="list-group fixed-content" id="sidenav">
        <ul class="navbar-links">
          <li class="navbar-dropdown list-group-item list-group-item-action">
            <a href="#head1">Overview</a>
          </li>
          <li class="navbar-dropdown list-group-item list-group-item-action">
            <a href="#head2">Engineering Cycle #1</a>
          </li>
          <li class="navbar-dropdown list-group-item list-group-item-action">
            <a href="#head3">Engineering Cycle #2</a>
          </li>
          <li class="navbar-dropdown list-group-item list-group-item-action">
            <a href="#head4">Engineering Cycle #3</a>
          </li>
          <li class="navbar-dropdown list-group-item list-group-item-action">
            <a href="#head5">Engineering Cycle #4</a>
          </li>
          <li class="navbar-dropdown list-group-item list-group-item-action">
            <a href="#ref">References</a>
          </li>
        </ul>
      </ul>
    </div>



    <div data-bs-target="sidenav" data-bs-spy="scroll" class="col-9" style="padding-top: 20px" >
      <div class="container " data-bs-spy="scroll"  data-bs-offset="-200px">

        <!--intro-->
        <h1 style="color: #002147">Introduction</h1>
        <br>
        <p>
          The Design–Build–Test–Learn (DBTL) cycle is a framework used to derive designs that meet certain predefined requirements. The recursive nature of the DBTL cycle enables repeated improvements, and is a popular approach to building parts in synthetic biology. Each stage in the cycle has a distinct objective:
          <ul>
            <li>The design is a conceptualization of the engineered system.</li>
            <li>The build is the execution of the design, in the form of new DNA parts or engineered organisms.</li>
            <li>The test is an objective and unbiased measure of the performance of the build.</li>
            <li>The results of the test enable learning, which facilitates improved design.</li>
          </ul>
        </p>
        <p>
          Our DBTL cycles focussed on (1) finding an appropriate <i>E. coli</i> chassis for OMV production (2) attenuating ClyA for usage in synthetic biology (3) finding epitope tags for recombinant protein detection.
          <a id="head2"></a>
        </p>
        <br>

        <div>
          <h1 style="color: #002147">Selecting a hypervesiculating <i>E. coli</i> strain for OMV production</h1>
          <h5>Engineering Cycle #1</h5>
          <br>
          <p>
            <i>E. coli</i> K-12 lab strains naturally secrete OMVs, however the low rate of secretion makes OMV isolation from <i>E. coli</i> K-12 cultures very resource intensive. Utilizing <i>E. coli</i> strains optimized for OMV isolation reduces the culture volume needed as well as the total time and equipment required for downstream processing.
          </p>
          <h3>Design</h3>
          <br>
          <p>
             In order to improve OMV yield, certain gene mutations can be introduced which increase OMV production. Deletions in the following genes have shown to increase yield:
          </p>
          <br>
          <p>
            <strong>ΔtolR</strong>
            <br>
            In <i>E. coli</i>, the Tol-Pal system comprises five proteins, including three inner membrane proteins (TolA, TolQ, and TolR), one periplasmic protein TolB, and outer membrane protein Pal. The Tol-Pal system tethers the outer membrane to the peptidoglycan layer and the inner membrane. Deletion of tolR produces defects in the Tol-Pal system, causing the dissociation of outer membrane from the underlying peptidoglycan, inducing membrane curvature and blebbing <a href="ref">[1]</a>.
            <br>
            <br>
            <strong>ΔtolA</strong>
            <br>
            tolA is another protein in the Tol-Pal system. Deleting TolA creates a hypervesiculating phenotype in a manner similar to deleting tolR. <a href="">[2]</a>
            <br>
            <br>
            <strong>ΔdegP</strong>
            <br>
            degP is a periplasmic protease/ chaperone that processes misfolded/unfolded proteins in the periplasm. Deleting DegP enhances OMV production as a means to remove misfolded proteins from the cell. <a href="ref">[3]</a>
            <br>
            <br>
            <strong>Δnlp1</strong>
            <br>
            nlp1 is an outer membrane-anchored lipoprotein that has been associated with cell division, but its precise function is unknown. Deleting Nlp1 is thought to induce instability in the peptidoglycan-outer membrane cross-links, hence increasing OMV production. <a href="ref">[4]</a>
          </p>
          <br>

          <h3>Build</h3>
          <br>
          <p>
            Four <i>E. coli</i> K12 BW25113 strains, each with a single knockout of the genes (ΔtolR, ΔtolA, ΔdegP and Δnlp1) were obtained from the Keio collection <a href="ref">[5]</a>. These knockout strains contain a kanamycin resistance cassette in place of the genes.
          </p>
          <br>

          <h3>Test</h3>
          <br>
          <p>
            <strong>Comparing fitness</strong>
            <br>
            Given that our chosen <i>E. coli</i> chassis would experience a metabolic burden during heterologous protein production, we compared growth curves and final OD600 of the strains to gain a rough estimate of the fitness cost of the gene knockouts. All cultures were standardized to OD600 = 0.001 and grown for 18h with orbital shaking in a 96-well plate. OD600 was measured every 15 min by a microplate reader.
          </p>
          <div class="row">
              <img class="img" src="https://static.igem.wiki/teams/4359/wiki/human-practices/1.png" alt="picture">
              <p class="article">
                <em class="figure-legend"><strong>Fig 1: </strong>
                   (A) Comparative analysis of growth curves of four hypervesiculating <i>E. coli</i> strains. Individual growth curves of (B) ΔtolA (C) ΔtolR (D) ΔdegP and (E) Δnlp1. Error bars indicate mean ± standard error of mean of three biological replicates measured in duplicate.
                </em>
              </p>
          </div>
          <br>
          <p>
            <strong>Table 1:</strong> Final OD600 of four hypervesiculating strains. Values are reported as mean ± standard error of mean of three biological replicates, measured in duplicate.
          </p>
          <br>
          <table class="table">
            <thead>
              <tr>
                <th scope="col" style="text-align: center">Strain</th>
                <th scope="col" style="text-align: center">Mean final OD600 ± standard error of mean</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td style="text-align: center">ΔtolA</td>
                <td style="text-align: center">0.71 ± 0.015</td>
              </tr>
              <tr>
                <td style="text-align: center">ΔtolR</td>
                <td style="text-align: center">0.71± 0.022</td>
              </tr>
              <tr>
                <td style="text-align: center">Δnlp1</td>
                <td style="text-align: center">0.70 ± 0.015</td>
              </tr>
              <tr>
                <td style="text-align: center">ΔdegP</td>
                <td style="text-align: center">0.67 ± 0.012</td>
              </tr>
            </tbody>
          </table>
          <br>
          <br>
          <p>
            <strong>Comparing total OMV yield</strong>
            <br>
            The concentration of OMVs in a solution can be determined by nanoparticle tracking analysis (NTA). Unfortunately we did not have access to NTA. OMVs can be indirectly quantified using protein concentration as a proxy measure <a href="ref">[2][6]</a>. Protein quantification was performed via a Bradford Assay. A standard curve was constructed using BSA and the regression line was used to compute protein concentrations of OMV samples (Table 2, Fig 2).
            <br>
            <br>
          </p>
          <p>
            <strong>Table 2:</strong> Protein concentration of OMVs fractions isolated from ΔtolR and Δnlp1. Higher total protein was noted for the OMV fraction isolated from Δnlp1. Measurements were carried out in duplicate.
            <br><br>
            <table class="table">
              <thead>
                <tr>
                  <th scope="col" style="text-align: center">Sample</th>
                  <th scope="col" style="text-align: center">Mean adjusted absorbance at 595nm (A)</th>
                  <th scope="col" style="text-align: center">Protein concentration (ug/ml) (C)</th>
                  <th scope="col" style="text-align: center">Total volume (ml) (T)</th>
                  <th scope="col" style="text-align: center">Total protein (P = C*T) (ug)</th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td style="text-align: center">ΔtolR OMV</td>
                  <td style="text-align: center">0.028</td>
                  <td style="text-align: center">100.64</td>
                  <td style="text-align: center">0.11</td>
                  <td style="text-align: center">11.06993</td>
                </tr>
                <tr>
                  <td style="text-align: center">Δnlp1 OMV</td>
                  <td style="text-align: center">0.011</td>
                  <td style="text-align: center">48.18</td>
                  <td style="text-align: center">0.31</td>
                  <td style="text-align: center">14.93495</td>
                </tr>
              </tbody>
            </table>
          </p>
          <div class="row">
              <img class="img" src="https://static.igem.wiki/teams/4359/wiki/engineering/eng2.png" alt="picture">
              <p class="article">
                <em class="figure-legend"><strong>Fig 2: </strong>
                   Comparative analysis of the OMV yield from ΔtolR and Δnlp1. OMV yield is measured as total protein in the sample.
                </em>
              </p>
          </div>
          <br>
          <p>
            <strong>Comparing the size distribution of OMVs</strong>
            <br>
            The size distribution of nanoparticles such as OMVs can be measured via dynamic light scattering. We compared the size distributions of OMV isolated from <i>E. coli</i> ΔtolR and Δnlp1 (Fig 3)
            <div class="row">
                <img class="img" src="https://static.igem.wiki/teams/4359/wiki/engineering/eng3.png" alt="picture">
                <p class="article">
                  <em class="figure-legend"><strong>Fig 3: </strong>
                     Comparison of the size distribution of OMVs isolated from <i>E. coli</i> ΔtolR and <i>E. coli</i> Δnlp1. The X axis indicates diameter (in nm) and the Y axis indicates the percentage. There is a strong correspondence between the two distributions.
                  </em>
                </p>
            </div>
          </p>
          <br>
          <p>
            Average diameter of <i>E. coli</i> ΔtolR OMVs = 40 nm
            <br>
            Average diameter of <i>E. coli</i> Δnlp1 OMVs = 39 nm
          </p>
          <br>

          <h3>Learn</h3>
          <br>
          <p>
            Among the four strains, ΔdegP showed delayed growth kinetics and lowest final OD600 (Fig 1, Table 1) and was considered unsuitable for heterologous protein expression. Literature suggests that ΔtolR produces more OMVs than ΔtolA [7]. Therefore, further analysis of OMV yield and size distribution was restricted to ΔtolR and Δnlp1. We found no significant difference between the sizes of OMVs derived from both strains (Fig 3), but found that Δnlp1 produced more OMVs than ΔtolR when subjected to the same culture conditions and isolation protocol (Fig 2). From our observations, we chose  Δnlp1 to produce OMVs for the remainder of the project.
            <a id="head3"></a>
          </p>
        </div>
        <br>

        <div>
          <h1 style="color: #002147">Attenuating Cytolysin A</h1>
          <h5>Engineering Cycle #2</h5>
          <br>
          <p>
            Cytolysin A (ClyA) is a 34 kDa protein coded for by the ClyA gene in <i>E. coli</i> genome. ClyA preferentially localizes to the outer membrane, causing it to be enriched in OMVs. We exploited this property to use ClyA to localize and display recombinant fusion partners on the surface of OMV. The existing version of ClyA on the registry, (<a href="">BBa_K811000</a>) is hemolytically active and toxic to mammalian cells [8]. It has been used by teams in other composite parts, such as <a href="http://parts.igem.org/Part:BBa_K3989010">BBa_K3989010</a>, and </a href="http://parts.igem.org/Part:BBa_K4375019">BBa_K4375019</a>. While genetic fusions with other proteins abolishes the pore-forming ability of ClyA, we wished to create a safer alternative for future teams to work with.
          </p>
          <br>

          <h3>Design</h3>
          <br>
          <p>
            ClyA exists in three known conformations: monomeric, protomeric and oligomeric. The monomer is the soluble form of the protein that undergoes a conformational change to the membrane-inserted protomeric form. The conformational change involves rearrangement of the beta-tongue motif to a helix-turn-helix motif. Upon reaching the target membrane, ClyA protomers oligomerize to insert into the membrane to form a pore. We designed ClyA (Y181F) (<a href="http://parts.igem.org/Part:BBa_K4359001">BBa_K4359001</a>), a derivative of ClyA with a Y181F substitution mutation. This tyrosine residue is contained in the cholesterol binding site in the beta-tongue motif of ClyA. The substitution is speculated to hinder the transition from the beta-tongue motif to the helix-turn-helix motif accompanying protomer formation. Y181F results in a near loss of the hemolytic activity of ClyA, as determined by erythrocyte lysis assays in previous studies [9].
          </p>
          <br>

          <h3>Build</h3>
          <br>
          <p>
            We avoided working with the unattenuated version of ClyA. We wished to evaluate the safety of ClyA (Y181F) when used as intended in a composite part like ClyA-3XFLAG-THP (<a href="http://parts.igem.org/Part:BBa_K4359009">BBa_K4359009</a>). We obtained the sequence for ClyA-3XFLAG-THP via gene synthesis.
          </p>
          <br>

          <h3>Test</h3>
          <br>
          <p>
            The MTT assay is a colorimetric assay for assessing cell metabolic activity, and can be used to measure cell viability. In this case, we aimed to measure cell viability following treatments of different concentrations of THP-OMVs expressing ClyA-3XFLAG-THP, as compared to a control treatment of PBS. SK-BR-3 cells utilised for this assay.The mean percentage viability is reported as the viability of THP-treated SKBR3 cells normalized to the mean viability following PBS treatment (Fig 4). The complete protocol can be found on the <a href="https://2022.igem.wiki/iiser-tvm/experiments">experiments page</a>.
          </p>
          <div class="row">
              <img class="img" src="https://static.igem.wiki/teams/4359/wiki/engineering/eng4.png" alt="picture">
              <p class="article">
                <em class="figure-legend"><strong>Fig 4: </strong>
                   Cytotoxicity of THP-OMVs as measured using the MTT assay with SK-BR-3 cell cultures. Values are reported as mean ± standard error of mean of three biological replicates.
                </em>
              </p>
          </div>
          <br>

          <h3>Learn</h3>
          <br>
          <p>
            The results of the MTT assay indicated that THP-OMVs did not significantly affect cell viability. We utilized ClyA (Y181F) for our other construct ClyA-V5-Affi (<a href="http://parts.igem.org/Part:BBa_K4359008">BBa_K4359008</a>).
          </p>
          <a id="head4"></a>
        </div>
        <br>

        <div>
        <h1 style="color: #002147">Selecting an epitope tag for detection of recombinant proteins</h1>
        <h5>Engineering Cycle #3</h5>
        <br>

        <h3>Design</h3>
        <br>
        <p>
          Detecting our recombinant proteins was an important part of the project. Epitope tags can be used for this purpose as they are well characterized and antibodies are commercially available. We required two tags whose detection was orthogonal and whose antibodies would cross react minimally with mammalian proteins. We originally received the construct ClyA-Myc-Affi (<a href="http://parts.igem.org/Part:">BBa_K4359006</a>), containing the myc tag. We hypothesized that antibodies against the myc tag would also cross react with endogenous c-myc which is often overexpressed in many cancer cell lines.
        </p>
        <br>

        <h3>Build</h3>
        <br>
        <p>
          ClyA-Myc-Affi was obtained from Dr. Vipul Gujrati, TUM Germany. It was cloned in the backbone pGEX-4T1.
        </p>
        <br>

        <h3>Test</h3>
        <br>
        <p>
          We performed a Western blot of cell lysate of the breast cancer cell line SK-BR-3 (Fig 5). Endogenous c-myc is 62 kDa but is known to degrade quickly to lower molecular weights.
        </p>
        <div class="row">
            <img class="img" src="https://static.igem.wiki/teams/4359/wiki/engineering/eng5.png" alt="picture">
            <p class="article">
              <em class="figure-legend"><strong>Fig 5: </strong>
                 Western blot of SK-BR-3 cell lysate. The membrane was probed with monoclonal anti-myc tag antibody.
              </em>
            </p>
        </div>
        <br>

        <h3>Learn</h3>
        <br>
        <p>
          While we were unable to ascertain whether the signal detected was from c-myc, we concluded that the anti-myc tag antibody cross-reacted with components of the cell lysate of our test cell line, SK-BR-3. This could potentially interfere with detection during cellular uptake studies. We designed an alternative version with a V5 tag in place of the myc tag, ClyA-V5-Affi (<a href="http://parts.igem.org/Part:BBa_K4359008">BBa_K4359008</a>)
          <a id="ref"></a>
        </p>
        <br>
      </div>

      <h1 style="color: #002147">References</h1>
      <br>
      <ol>
        <li>Shrivastava, R., Jiang, X., & Chng, S.-S. (2017). Outer membrane lipid homeostasis via retrograde phospholipid transport in Escherichia coli: A physiological function for the Tol-Pal complex. Molecular Microbiology, 106(3), 395–408. <a href="https://doi.org/10.1111/mmi.13772">https://doi.org/10.1111/mmi.13772</a></li>
        <br>
        <li>Reimer, S. L., Beniac, D. R., Hiebert, S. L., Booth, T. F., Chong, P. M., Westmacott, G. R., Zhanel, G. G., & Bay, D. C. (2021). Comparative Analysis of Outer Membrane Vesicle Isolation Methods With an Escherichia coli tolA Mutant Reveals a Hypervesiculating Phenotype With Outer-Inner Membrane Vesicle Content. Frontiers in Microbiology, 12, 628801. <a href="https://doi.org/10.3389/fmicb.2021.628801">https://doi.org/10.3389/fmicb.2021.628801</a></li>
        <br>
        <li>Schwechheimer, C., & Kuehn, M. J. (2013). Synthetic Effect between Envelope Stress and Lack of Outer Membrane Vesicle Production in Escherichia coli. Journal of Bacteriology, 195(18), 4161–4173. <a href="https://doi.org/10.1128/JB.02192-12">https://doi.org/10.1128/JB.02192-12</a></li>
        <br>
        <li>Schwechheimer, C., Rodriguez, D. L., & Kuehn, M. J. (2015). NlpI‐mediated modulation of outer membrane vesicle production through peptidoglycan dynamics in Escherichia coli. MicrobiologyOpen, 4(3), 375–389. <a href="https://doi.org/10.1002/mbo3.244">https://doi.org/10.1002/mbo3.244</a></li>
        <br>
        <li><i>E. coli</i> Keio Knockouts. Link <a href="https://horizondiscovery.com/en/non-mammalian-research-tools/products/e-coli-keio-knockouts">here</a></li>
        <br>
        <li>Gujrati, V., Kim, S., Kim, S.-H., Min, J. J., Choy, H. E., Kim, S. C., & Jon, S. (2014). Bioengineered Bacterial Outer Membrane Vesicles as Cell-Specific Drug-Delivery Vehicles for Cancer Therapy. ACS Nano, 8(2), 1525–1537. <a href="">https://doi.org/10.1021/nn405724x</a></li>
        <br>
        <li>Li, Q., Li, Z., Fei, X., Tian, Y., Zhou, G., Hu, Y., Wang, S., & Shi, H. (2022). The role of TolA, TolB, and TolR in cell morphology, OMVs production, and virulence of Salmonella Choleraesuis. AMB Express, 12(1), 5. <a href="">https://doi.org/10.1186/s13568-022-01347-4</a></li>
        <br>
        <li>Mueller, M., Grauschopf, U., Maier, T., Glockshuber, R., & Ban, N. (2009). The structure of a cytolytic α-helical toxin pore reveals its assembly mechanism. Nature, 459(7247), 726–730. <a href="https://doi.org/10.1038/nature08026">https://doi.org/10.1038/nature08026</a></li>
        <br>
        <li>Kulshrestha, A., Maurya, S., Gupta, T., Roy, R., Punnathanam, S., & Ayappa, K. G. (2021). Conformational flexibility is a key determinant of the lytic activity of the pore forming protein, Cytolysin A [Preprint]. Biophysics. <a href="https://doi.org/10.1101/2021.10.23.465544">https://doi.org/10.1101/2021.10.23.465544</a></li>
      </div>
    </div>
  </div>
</div>

    <script  src="../../static/carousel.js"></script>
    <script src='../../static/style/modalscript.js'></script>
    <script  src="../../static/style/modaljs.js"></script>


<!-- Footer: MUST mention license AND have a link to team wiki's repository on gitlab.igem.org -->
<footer class="pt-5 pb-5 footer py-5 bg-dark text-white">
  <div class="container">
    <div class="row mb-4">
      <div class="col-lg-6 col-xs-12">
        <h4 class="mb-3">Heading</h4>
        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ac ante mollis quam tristique convallis</p>
      </div>
      <div class="col-lg-3 col-xs-12">
        <h4 class="mt-lg-0 mt-sm-3">Links</h4>
          <ul class="m-2 p-2">
            <li><a href="#">School of Biology, IISER TVM</a></li>
            <li><a href="#">Instagram</a></li>
            <li><a href="#">Twitter</a></li>
            <li><a href="#">YouTube</a></li>
            <li><a href="#">iGEM 2021</a></li>
          </ul>
      </div>
      <div class="col-lg-3 col-xs-12">
        <h4 class="mt-lg-0 mt-sm-4 mb-3">Contact</h4>
        <p>IISER Thiruvananthapuram, Maruthamala PO, Vithura, Kerala, India - 695551</p>
        <p class="mb-0">+91 0471 277 8009</p>
        <p>igem@iisertvm.ac.in</p>
      </div>
    </div>
    <hr>
    <!-- The following MUST be on every page: license information and link to the repository on gitlab.igem.org -->
    <div class="row mt-4">
      <div class="col">
        <p class="mb-0"><small>© 2022 - Content on this site is licensed under a <a class="subfoot" href="https://creativecommons.org/licenses/by/4.0/" rel="license">Creative Commons Attribution 4.0 International license</a>.</small></p>
        <p><small>The repository used to create this website is available at <a href="https://gitlab.igem.org/2022/iiser-tvm">gitlab.igem.org/2022/iiser-tvm</a>.</small></p>
      </div>
    </div>
  </div>
</footer>

<!-- Wiki Tools: Teams are allowed to remove it -->
<style>
    .wiki-tools-footer {
        --bs-gutter-x: 1.5rem;
        background: #222124;
        margin: 0;
        max-width: 100%;
        padding: 1rem 3rem;
        display: flex;
        gap: 1.5em;
    }
    .wiki-tools-footer a {
        text-decoration: none;
        color: white;
        font-weight: 400;
        transition: color 0.3s;
    }
    .wiki-tools-footer a:hover {
        color: #08C784;
    }
    .wiki-tools-footer img {
        object-fit: contain;
    }
    header{
    text-align: center;
    text-transform: capitalize;
    background-color: #084bc7;
    height: 10%;
    padding: 1%;
  }
</style>

<section class="wiki-tools-footer container">
    <a id="wiki-tools-link">Access Wiki Tools</a>
    <a id="wiki-tools-repo">View Source Code</a>
    <img id="wiki-tools-pipeline">
</section>

<script>
    const wikiToolsLink = document.getElementById('wiki-tools-link')
    const wikiToolsRepo = document.getElementById('wiki-tools-repo')
    const wikiToolsPipeline = document.getElementById('wiki-tools-pipeline')

    const path = document.location.pathname.split('/')

    const competitionSlug = document.location.host.split('.')[0]
    const teamSlug = path[1]
    const page = path[2] === '' ? 'index' : path[2]

    wikiToolsLink.href = `https://tools.igem.org/wiki-tools/${competitionSlug}/${teamSlug}/${page}`
    wikiToolsRepo.href = `https://gitlab.igem.org/${competitionSlug}/${teamSlug}`
    wikiToolsPipeline.src = `https://gitlab.igem.org/${competitionSlug}/${teamSlug}/badges/main/pipeline.svg`
</script>

<!--scrollspy-->
<script>
  var scrollspy= new bootstrap.ScrollSpy(document.body, {target: "#sidenav"})
</script>


<!-- Bootstrap Bundle with Popper -->
<!--<script src="{{ url_for('static', filename = 'bootstrap.bundle.min.js') }}"></script>-->
<script src="../../static/bootstrap.bundle.min.js"></script>

</body>
</html>
